ch_pipeline/processing/daily.py

```diff
@@ -968,28 +968,32 @@ def _available_tags(self) -> list:
 
         This includes any that currently exist or are in the job queue.
         """
-        # Get all desired csds from the config file
-        csds = self._all_tags
-        # Automatically exclude bad days
-        csds = [csd for csd in csds if csd not in self._exclude]
-        csds_sorted = sorted(csds)
-        # Get a list of CSDS whose files are entirely available online
-        csds_to_run = available_csds(
-            csds_sorted,
-            pad=self._padding,
-            required_coverage=self._min_coverage,
-            weather_coverage=self._weather_coverage,
-        )
+        csds = self.runnable_tags(require_online=True)
 
-        return [f"{csd:.0f}" for csd in csds if csd in csds_to_run]
+        return [f"{csd:.0f}" for csd in csds]
 
     @property
-    def _all_tags(self) -> list:
+    def _config_tags(self) -> list:
         """Return all tags desired from the config."""
         return unique_ordered(
             csd for i in self._intervals for csd in expand_csd_range(*i)
         )
 
+    def runnable_tags(self, require_online=False) -> list:
```

Comment:
It's odd that `require_online` defaults to False here, but in `available_csds` below, it defaults to True.  I'd make the defaults the same (one way or the other).

Reply:
Ah good call